src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -112,6 +112,11 @@ public class GenerateScriptCommand : Command
             {
                 IsRequired = false
             };
+            var targetApiUrl = new Option<string>("--target-api-url")
+            {
+                IsRequired = false,
+                Description = "The URL of the target API, if not migrating to github.com (default: https://api.github.com). It will be ignored for ADO to GitHub migration path."
```

Comment:
Do I need to explicitly say that it will be ignored for ADO migration path?

Reply:
I wouldn't, especially because we're about to remove the ADO migration path from this tool.

Reply:
I would however mention that this is for GHAE

Reply:
addressed in a02ad5bc93a104ef3c5b7ee19bdd1fe2df9836bd